### PR TITLE
ref #304: style TCMSFieldText_ShowExportURL and add missing button te…

### PR DESCRIPTION
**Pull request by pixeljunkie**
…xt translation

| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#304
| License       | MIT

### CODE REVIEW

File: src/ShopProductExportBundle/Resources/translations/admin.de.xliff [hunk 1]
```diff
@@ -18,7 +18,10 @@
                 <source>chameleon_system_shop_product_export.field_show_export_url.error_invalid_field_owner</source>
                 <target>Fehler beim Auslesen des aktuellen Datensatzes.</target>
             </trans-unit>
-
+            <trans-unit id="chameleon_system_shop_product_export.field_show_export_url.show_list_button_title">
+                <source>chameleon_system_shop_product_export.field_show_export_url.show_list_button_title</source>
+                <target>Export URL Liste anzeigen</target>
```
Comment (umulmrum) [inline src/ShopProductExportBundle/Resources/translations/admin.de.xliff]:
```suggestion
                <target>Export-URL-Liste anzeigen</target>
```

File: src/ShopProductExportBundle/Resources/translations/admin.en.xliff [hunk 2]
```diff
@@ -18,7 +18,10 @@
                 <source>chameleon_system_shop_product_export.field_show_export_url.error_invalid_field_owner</source>
                 <target>Error while reading current record (invalid field owner).</target>
             </trans-unit>
-
+            <trans-unit id="chameleon_system_shop_product_export.field_show_export_url.show_list_button_title">
+                <source>chameleon_system_shop_product_export.field_show_export_url.show_list_button_title</source>
+                <target>Show Export URL List</target>
```
Comment (umulmrum) [inline src/ShopProductExportBundle/Resources/translations/admin.en.xliff]:
```suggestion
                <target>Show export URL list</target>
```

File: src/ShopProductExportBundle/Resources/translations/admin.de.xliff (hunk 1 above)
Comment (pixeljunkie) [reply src/ShopProductExportBundle/Resources/translations/admin.de.xliff]: done

File: src/ShopProductExportBundle/Resources/translations/admin.en.xliff (hunk 2 above)
Comment (pixeljunkie) [reply src/ShopProductExportBundle/Resources/translations/admin.en.xliff]: done

**Comment by umulmrum** [comment]
Thanks @pixeljunkie

**Pull request merged**